### PR TITLE
refactor(EventHandling): use interactions

### PR DESCRIPTION
**Pull request by J-Human**
**Please describe the changes this PR makes and why it should be merged:**
This makes use of the interaction event and replaces the message one with the former, this would be similar to the command handling counterpart.

### CODE REVIEW

File: code-samples/event-handling/file-setup/events/interactionCreate.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	name: 'interactionCreate',
+	execute(interaction) {
+		console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction`);
```
Comment (WiseDevHelper) [inline code-samples/event-handling/file-setup/events/interactionCreate.js:4]:
```suggestion
		console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction.`);
```

File: guide/event-handling/README.md [hunk 2]
```diff
@@ -14,22 +14,22 @@ client.once('ready', () => {
 	console.log('Ready!');
 });
 
-client.on('message', message => {
-	console.log(`${message.author.tag} in #${message.channel.name} sent: ${message.content}`);
+client.on('interactionCreate', interaction => {
+	console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction`);
```
Comment (WiseDevHelper) [inline guide/event-handling/README.md:18]:
```suggestion
	console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction.`);
```

**Review by WiseDevHelper — CHANGES_REQUESTED** [review]
LGTM, Just a small suggestion.

File: guide/event-handling/README.md [hunk 3]
```diff
@@ -43,9 +43,9 @@ module.exports = {
 
 ```js
 module.exports = {
-	name: 'message',
-	execute(message) {
-		console.log(`${message.author.tag} in #${message.channel.name} sent: ${message.content}`);
+	name: 'interactionCreate',
+	execute(interaction) {
+		console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction`);
```
Comment (WiseDevHelper) [inline guide/event-handling/README.md:48]:
```suggestion
		console.log(`${interaction.user.tag} in #${interaction.channel.name} triggered an interaction.`);
```

**Comment by J-Human** [comment]
Gotta add a v13/v12 folder for legacy users.

**Comment by J-Human** [comment]
\* ready

**Comment by J-Human** [comment]
All good!

**Comment by monbrey** [comment]
> 
> 
> Gotta add a v13/v12 folder for legacy users.

I dont think that's right. v11 and v12 content is going to be archived.

**Comment by J-Human** [comment]
> > Gotta add a v13/v12 folder for legacy users.
> 
> I dont think that's right. v11 and v12 content is going to be archived.

Ah, my bad then. I'll remove it.

**Comment by J-Human** [comment]
Ok, I'm really done.

**Review by WiseDevHelper — CHANGES_REQUESTED** [review]
`code-samples/event-handling/file-setup` -> `code-samples/event-handling/13/file-setup`

**Comment by J-Human** [comment]
> `code-samples/event-handling/file-setup` -> `code-samples/event-handling/13/file-setup`

See Monbrey's comment above.

**Comment by WiseDevHelper** [comment]
> > `code-samples/event-handling/file-setup` -> `code-samples/event-handling/13/file-setup`
> 
> See Monbrey's comment above.

Doesn't that refer to v12 folders?

**Comment by J-Human** [comment]
> > > `code-samples/event-handling/file-setup` -> `code-samples/event-handling/13/file-setup`
> > 
> > 
> > See Monbrey's comment above.
> 
> Doesn't that refer to v12 folders (only)?

Right. I'll do your suggestion.

**Comment by J-Human** [comment]
Perhaps it should be `.../file-setup/13/...` instead? Basing on the command handler part.

**Comment by J-Human** [comment]
Alright, done.

**Comment by WiseDevHelper** [comment]
> Perhaps it should be `.../file-setup/13/...` instead? Basing on the command handler part.

I guess that's fine but would like @Danktuary's input to confirm.

**Comment by J-Human** [comment]
> > Perhaps it should be `.../file-setup/13/...` instead? Basing on the command handler part.
> 
> I guess that's fine but would like @Danktuary's input to confirm.

Already did it anyway. 😆

**Review by Danktuary — APPROVED** [review]

**Pull request merged**